src/Misc/layoutbin/en-US/strings.json

```diff
@@ -182,6 +183,9 @@
     "Run as a Windows service that logs on as a domain account (Windows only)",
     ".\\config.cmd --unattended --url https://myaccount.visualstudio.com --auth pat --token myToken --pool default --agent myAgent --runAsService --windowsLogonAccount myDomain\\myUserName --windowsLogonPassword myPassword",
     "",
+    "Run as a Windows service that logs on as a domain account (Windows only) and doesn't start service immediately",
+    ".\\config.cmd --unattended --url https://myaccount.visualstudio.com --auth pat --token myToken --pool default --agent myAgent --runAsService --windowsLogonAccount myDomain\\myUserName --windowsLogonPassword myPassword",
```

Comment:
The `--preventServiceStart` arg is not provided here

Reply:
added